gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -185,6 +357,8 @@ message TripUpdate {
     // To specify a completely certain prediction, set its uncertainty to 0.
     optional int32 uncertainty = 3;
 
+    optional int64 scheduled_time = 4;
```

Comment:
I think this should be `schedule_time` (without the "d") - see [my comment in the Google Doc](https://docs.google.com/document/d/1oPfQ6Xvui0g3xiy1pNiyu5cWFLhvrTVyvYsMVKGyGWM/edit?disco=AAAAJHFVCFs)